Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN apk add --no-cache \
   && pip3 install awscli \
   && rm -rf /var/cache/apk/*
 
+RUN git --version
```

Comment:
This line was from the original eonent pull, it does no harm and it's helpful info